inventory/byo/hosts.example

```diff
@@ -78,5 +78,5 @@ ose3-etcd[1:3]-ansible.test.example.com
 
 # host group for nodes
 [nodes]
-ose3-master[1:3]-ansible.test.example.com openshift_scheduleable=False
+ose3-master[1:3]-ansible.test.example.com openshift_schedulable=False
```

Comment:
Ok, I checked both those repos and didn't find any instances of 'schedulea', so I think that may have been corrected already.